api/src/propagation/TextMapPropagator.ts

```diff
@@ -27,7 +27,7 @@ import { Context } from '../context/types';
  * usually implemented via library-specific request interceptors, where the
  * client-side injects values and the server-side extracts them.
  */
-export interface TextMapPropagator<Carrier = any> {
```

Comment:
Changing the default value of a parameter would be a breaking change. I'd suggest leaving this as is.